### PR TITLE
Ember CLI transfer (WIP)

### PR DESCRIPTION
**Pull request by hakubo**
I am checking if we can move to Ember CLI
Expect RFC soon 

This requires muuuch more work
and will be hard to finish on my own 

App still does not start
but it does compile :)

Creating this PR for visibility and for fun

### CODE REVIEW

**Comment by rogatty** [comment]
I like it! But `Files changed 1,680`? Forget about CR :D

**Comment by hakubo** [comment]
Ticket spent in CR: 1034days

File: app/components/base-media.js [hunk 1]
```diff
@@ -0,0 +1,128 @@
+import Ember from 'ember';
+import Thumbnailer from '../utils/modules/thumbnailer';
+import LinkedGalleryMediaComponent from 'linked-gallery-media';
+import GalleryMediaComponent from 'gallery-media';
+import VideoMediaComponent from 'video-media';
+import ImageMediaComponent from 'image-media';
+
+var baseMedia = Ember.Component.extend({
```
Comment (kenkouot) [inline app/components/base-media.js]: classes should be capitalized, even base ones

File: app/routes/application.js [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+            this.send('openLightbox', 'ads', { contents: contents });
+        };
+    },
+    actions: {
+        loading: function () {
+            //this.controller.showLoader();
+        },
+        didTransition: function () {
+            // Activate any A/B tests for the new route
+			variantTesting.activate();
```
Comment (kenkouot) [inline app/routes/application.js]: off spacing

**Pull request closed without merging**